interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+    mount options=(rw, bind) /dev/{,pts/}ptmx -> /dev/{,pts/}ptmx,
+
+    # apparmor="DENIED" operation="mount" info="failed mntpnt match" error=-13 profile="snap.anbox.container-manager//lxc" name="/" pid=4539 comm="anbox" flags="rw, rslave"
+    umount /,
+    umount /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+    # done post pivot_root
+
+    # starting processes in the container
+    /anbox-init.sh ixr,
+    @{PROC}/1/attr/current r,
+
+    # end processes in the container
+
+    owner @{PROC}/@{pid}/net/unix r,
+    unix (bind,listen) type=stream addr="@/var/snap/anbox/common/containers/default/command",
+    /sys/fs/cgroup/*/lxc/{,**} rw,   # this should be anbox specific
+    @{PROC}/@{pid}/uid_map rw,
+    @{PROC}/@{pid}/gid_map rw,
+    /sys/fs/cgroup/cpuset/cpuset.cpus r,
+    /sys/fs/cgroup/cpuset/cpuset.mems r,
+
+    # None of these work with this denial:
+    # operation="ptrace" profile="snap.anbox.container-manager" pid=27006 comm="anbox" requested_mask="trace" denied_mask="trace" peer="snap.anbox.container-manager"
+    ptrace peer=@{profile_name},
+    #ptrace peer=snap.@{SNAP_NAME}.*,
+    #ptrace,
+
+    capability chown,
+    capability mknod,
+    capability fowner,
+    capability fsetid,
+    capability dac_override,
+    capability dac_read_search,
+    capability setuid,
+    capability setgid,
+    capability sys_admin,
+
+    # We need to allow access to the fuse device here as we're in a child
+    # profile and fuse-support wouldn't help at this point.
+    /dev/fuse rw,
+
+    # container start
+    owner @{PROC}/[0-9]*/fd/ r,
+    owner @{PROC}/[0-9]*/stat r,
+    owner @{PROC}/[0-9]*/ns/ r,
+    /dev/ptmx rw,
+    /dev/pts/[0-9]* rw,
+    capability net_admin,
+    /sys/devices/** r,
+    capability sys_ptrace,
+    / r,
+    capability sys_module, # why?
+    capability kill,
+
+    # Allow liblxc to change to the container child profile
+    # FIXME: Can we get the parent profile name somehow?
+    change_profile unsafe /** -> snap.anbox.container-manager//container,
+
+    mount options=(rw,bind),
+    mount options=(rw,rbind),
+    mount options=(rw,make-slave) -> **,
+    mount options=(rw,make-rslave) -> **,
+    mount options=(rw,make-shared) -> **,
+    mount options=(rw,make-rshared) -> **,
+    mount options=(rw,make-private) -> **,
+    mount options=(rw,make-rprivate) -> **,
+    mount options=(rw,make-unbindable) -> **,
+    mount options=(rw,make-runbindable) -> **,
+}
+
+# The container child profile is used by the actual Android container the Anbox container
+# manager creates and manages through liblxc. The policy here is inheriting from the one
+# LXD defines for its containers. See https://github.com/lxc/lxd/blob/master/lxd/apparmor.go
+profile "container" flags=(attach_disconnected,mediate_deleted) {
+    ### Base profile
+    capability,
+    dbus,
+    file,
+    network,
+    umount,
+
+    # Allow us to receive signals from anywhere.
+    signal (receive),
```

Comment:
Please adjust this to be:
```
signal (receive) peer=unconfined,
signal (receive) peer=snap.*,
signal (receive) peer=%s,
```

where '%s' is whatever profile name would send this signals (eg, the snap.anbox.container-manager). Note the 'snap.*' requires the sender to have process-control.